src/admin.ts

```diff
@@ -61,16 +61,22 @@ admin.post("/create", async (req, res) => {
   return res.sendStatus(204);
 });
 
-admin.post("/follow/:actor", async (req, res) => {
-  const actor: string = req.app.get("actor");
 
-  const object = req.params.actor;
-  const uri = `https://${HOSTNAME}/@${crypto.randomUUID()}`;
-  await send(actor, object, {
+admin.post("/follow/:actor/:hostname/:port/:proto", async (req, res) => {
```

Comment:
I had to change this endpoint, it was 404ing every time for me without this change